src/status_im/ui/components/connectivity/view.cljs

```diff
@@ -78,98 +73,113 @@
 
 (defn connectivity-status
-  [{:keys [connected?]} status-hidden]
-  (let [anim-translate-y (animation/create-value neg-connectivity-bar-height)
-        anim-opacity     (animation/create-value 0)]
+  [{:keys [connected?]} status-hidden height]
+  (let [neg-connectivity-bar-height (- @height)
+        anim-translate-y            (animation/create-value neg-connectivity-bar-height)
+        anim-opacity                (animation/create-value 0)]
     (reagent/create-class
-     {:component-did-mount
-      (fn []
-        (manage-visibility connected? false
-                           anim-opacity anim-translate-y status-hidden))
-      :should-component-update
-      ;; ignore :loading-indicator?
+     {:component-did-mount     (fn []
+                                 (manage-visibility connected? false
+                                                    anim-opacity anim-translate-y status-hidden height))
+      :should-component-update ;; ignore :loading-indicator?
       (fn [_ [_ old_p] [_ new_p]]
         (not= (dissoc old_p :loading-indicator?)
               (dissoc new_p :loading-indicator?)))
-      :component-did-update
-      (fn [comp]
-        (manage-visibility (:connected? (reagent/props comp)) true
-                           anim-opacity anim-translate-y status-hidden))
-      :reagent-render
-      (fn [{:keys [message on-press-event connected? connecting?] :as opts}]
-        (when-not @status-hidden
-          [react/animated-view {:style               (styles/text-wrapper
-                                                      (assoc opts
-                                                             :height connectivity-bar-height
-                                                             :background-color (if connected?
-                                                                                 colors/green
-                                                                                 colors/gray)
-                                                             :transform anim-translate-y
-                                                             :opacity anim-opacity))
-                                :accessibility-label :connection-status-text}
-           (when connecting?
-             [react/activity-indicator {:color colors/white :margin-right 6}])
-           (if (= message :mobile-network)
-             [react/nested-text {:style    styles/text
-                                 :on-press (when on-press-event #(re-frame/dispatch [on-press-event]))}
-              (i18n/label :t/waiting-for-wifi) " "
-              [{:style {:text-decoration-line :underline}}
-               (i18n/label :t/waiting-for-wifi-change)]]
-             (when message
-               [react/text {:style    styles/text
-                            :on-press (when on-press-event #(re-frame/dispatch [on-press-event]))}
-                (i18n/label message)]))]))})))
+      :component-did-update    (fn [comp]
```

Comment:
but cljfmt doesn't do that, we try to keep the length of the lines under 80 chars (soft limit), for this reason in cases like this we put the key and value on separate lines